libs/designsystem/src/lib/components/icon/icon.component.ts

```diff
@@ -1,9 +1,15 @@
 import { Component, HostBinding, Input, OnChanges, SimpleChanges } from '@angular/core';
 
-import { kirbyIconSettings } from './kirby-icon-settings';
-import { Icon } from './icon-settings';
 import { IconRegistryService } from './icon-registry.service';
+import { Icon } from './icon-settings';
+import { kirbyIconSettings } from './kirby-icon-settings';
 
+export enum IconSizes {
+  XS = 'xs',
+  SM = 'sm',
+  MD = 'md',
+  LG = 'lg',
+}
 @Component({
```

Comment:
Consider readability:
```suggestion
}

@Component({
```